docker-multinode/common.sh

```diff
@@ -274,9 +269,9 @@ kube::multinode::restart_docker(){
       else
         DOCKER_CONF="/etc/default/docker"
         
-        kube::helpers::file_replace_line ${DOCKER_CONF} `# Replace content in this file` \
-          "--bip" `# Find a line with this content...` \
-          "OPTIONS=\"\$OPTIONS --mtu=${FLANNEL_MTU} --bip=${FLANNEL_SUBNET}\"" `# ...and replace the found line with this line`
+        kube::helpers::file_replace_line ${DOCKER_CONF} \ # Replace content in this file
```

Comment:
Oops, thanks for catching. That's a rebase conflict, I didn't explicitely add it now, but having the comment on the same line makes sense